go/kbfs/libkbfs/init.go

```diff
@@ -835,17 +835,12 @@ func doInit(
 	if err != nil {
 		log.CWarningf(ctx,
 			"Could not initialize block metadata store: %+v", err)
-		return nil, err
-		// TODO (KBFS-3659): when we can open levelDB read-only, re-enable
-		//                   this, instead of failing the init.
-		/*
-			notification := &keybase1.FSNotification{
-				StatusCode:       keybase1.FSStatusCode_ERROR,
-				NotificationType: keybase1.FSNotificationType_INITIALIZED,
-				ErrorType:        keybase1.FSErrorType_DISK_CACHE_ERROR_LOG_SEND,
-			}
-			defer config.Reporter().Notify(ctx, notification)
-		*/
+		notification := &keybase1.FSNotification{
```

Comment:
I don't think we need the notification -- this will lead to a confusing toast popup for the user.  I think you can keep it commented out.